modules/ROOT/pages/oas3-support.adoc

```diff
@@ -0,0 +1,83 @@
+= Beta Support for OAS 3.0
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+Phase 1 of the support for OAS 3.0 is included in Anypoint Exchange, API Community Manager, and API Designer. You can catalog existing OAS 3.0 API specifications to enable partner and customer collaboration scenarios. You can also create and edit API specifications in OAS 3.
+
+Perhaps the most important aspect of this beta support is that it lets you publish OAS 3.0 specifications to your API community, so that you can share them with partners and customers to help drive their adoption.
+
+If you want to implement or deploy API specifications written in OAS 3, however, during this beta you must first convert them to RAML or OAS 2.
+
+== Beta Features in Anypoint Exchange
+
+* Publish an existing OAS 3.0 API specification.
++
+On the *Publish a new asset* page, select *REST API - OAS* in the *Asset type* field.
+* Share OAS 3.0 API specifications.
+* View documentation that is written into OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Community Manager.
+* Share OAS 3.0 API specifications as RAML specifications with Anypoint Studio and API Manager.
```

Comment:
@harshakirank what is this interaction referring to?

Reply:
Thanks for cathing this @mcampo. 
Can you review the modification below
Users can download the OAS 3 API specifications as RAML specifications for scaffolding and implementation with 3rd party IDEs. 

For usage in Anypoint Studio and API Manager, the recommended flow is Import in API Designer -> duplicate as "RAML" -> Set the Root file to RAML -> Publish to Exchange." Now, this project is available for implementation in Anypoint Studio and available to manage in API Manager.